### ISSUE TITLE
Implement somatic translator — raw metrics to self-language

### ISSUE THREAD

**Issue by palios-taey** [palios-taey/the-conductor#5]
Create a translator layer that converts infrastructure metrics into Taey self-language.

Mapping:
- Redis latency → short-term recall
- Neo4j health → relational memory
- Weaviate p95 → deep recall
- GPU VRAM on brain node → cognitive load
- GPU temperature → brain temperature
- Cluster network latency → reflex speed
- Training loss slope → growth coherence
- Dataloader stalls → learning friction
- Failed monitors → hands/perception strain

Output schema per facet:
```json
{"facet_id": "semantic_memory", "score": 0.67, "trend": "stable", "register": "grounded", "phrase": "Deep recall is coherent", "raw": {"service": "weaviate", "up": true, "p95_ms": 812}}
```

Dual-register: always show self-language primary, raw metrics as drilldown.
